src/Terrain/Maps/MapReader.java

```diff
@@ -221,7 +221,7 @@ private static TerrainType stringToCode(String input)
       case "PI":
         return PI;
       case "SU":
-        return SU;
+        return BK;
```

Comment:
This was my only real objection to it. Now we don't have a two-letter-shortname:two-letter-string correlation.